charts/bindplane/templates/configmap.yaml

```diff
@@ -0,0 +1,12 @@
+{{- if .Values.backend.postgres.sslsecret.name }}
```

Comment:
This configmap is mounted by the init container and executed to set permissions.